db/schema.sql

```diff
@@ -21,8 +21,11 @@ CREATE TABLE users
   sex VARCHAR(30) NOT NULL,
   yearOfBirth INTEGER NOT NULL,
   email VARCHAR(100),
+  phone VARCHAR(15),
   date TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  hash VARCHAR(64) NOT NULL
+  hash VARCHAR(64) NOT NULL,
+  emailContact BOOLEAN NOT NULL,
```

Comment:
A common convention is to name Boolean fields like a yes/no question. Also as mentioned in #210, database columns should not be abbreviated, and we shouldn't worry too much about using long column names if they're more descriptive.

I'd suggest `is_email_contact_consent_granted` and `is_sms_contact_consent_granted`.